docs-content/general/6_product-features/6_dashboards/7_sql-editor.md

```diff
@@ -0,0 +1,54 @@
+---
+title: SQL Editor
+slug: sql-editor
+createdAt: 2025-01-27T00:00:00.000Z
+updatedAt: 2025-01-27T00:00:00.000Z
+---
+
+The SQL editor lets you write custom `SELECT` queries to retrieve your highlight.io data and aggregate it in useful ways. This is an alternative to the query builder, useful when you need to do more complex transformations to your data. The output is displayed in the same charts and tables supported by the query builder.
+
+![SQL editor](/images/docs/graphing/sql_editor.png)
+
+## ClickHouse SQL
+Highlight.io metadata is stored in ClickHouse and the SQL editor supports the ClickHouse SQL dialect. For questions about supported syntax or functions, you can check out the ClickHouse [docs](https://clickhouse.com/docs/en/sql-reference).
```

Comment:
Nit: Should we add a comma here? Think it read a little weird without it.

"Highlight.io metadata is stored in ClickHouse, and the SQL editor supports the ClickHouse SQL dialect."